src/main/java/com/hms_networks/americas/sc/extensions/historicaldata/CircularizedFileException.java

```diff
@@ -0,0 +1,12 @@
+package com.hms_networks.americas.sc.extensions.historicaldata;
+/**
```

Comment:
Not sure if it's a hard requirement, but the convention is to include a blank line between package/imports/javadocs.

Reply:
changed